makarov ivan/site_grid.html

```diff
@@ -90,18 +102,20 @@
             .container {
                 width: 581px;
                 height: 438px;
+                margin: auto;   /*главный див будет по центру*/
                 display: grid;
                 grid-template-columns: 290px 40px 50px 29px 50px 29px 50px 43px;
                 grid-template-rows: 144px 96px 94px 104px;
                 background: url(pics/layer6.png) no-repeat center;
-                }
+            }
         </style>
+    </head>
     <body>
         <div class=container>
             <div class=picture_of_sun><p></p></div>
-            <div class="word_sunny"><span>Sunny</span></div>
```

Comment:
Try to use more abstract names for classes here and below. For example, in this case, you could've used `weather-condition`.